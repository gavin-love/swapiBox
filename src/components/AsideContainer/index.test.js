import React from "react";
import { AsideContainer } from "./index.js";

describe("AsideContainer", () => {
  let wrapper;
  let mockData;

  it("matches the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("should return opening scroll when pass film", () => {
    mockData = {
      title: "A NEW HOPE",
      releaseDate: "1977-05-25",
      openingCrawl: "IT'S A WAR"
    };

    wrapper = <AsideContainer film={mockData} />;

    expect(wrapper).toMatchSnapshot();
  });
});
