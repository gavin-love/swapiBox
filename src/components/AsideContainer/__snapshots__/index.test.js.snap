// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AsideContainer matches the snapshot 1`] = `undefined`;

exports[`AsideContainer should return opening scroll when pass film 1`] = `
<Unknown
  film={
    Object {
      "openingCrawl": "IT'S A WAR",
      "releaseDate": "1977-05-25",
      "title": "A NEW HOPE",
    }
  }
/>
`;
